pipelines/bulk_rna_encode/bulk_rna_pipeline.wdl

```diff
@@ -0,0 +1,420 @@
+      File? madQCplot = mad_qc.madQCplot
+      File? mad_qc_python_log = mad_qc.python_log
+      Array[File] rna_qc_python_log = rna_qc.python_log
+      Array[File] rnaQC = rna_qc.rnaQC
+      Array[File] genes_results = rsem_quant.genes_results
+      Array[File] isoforms_results = rsem_quant.isoforms_results
+      Array[File] number_of_genes = rsem_quant.number_of_genes
+      Array[File] rsem_quant_python_log = rsem_quant.python_log
+    }
+}
+
+
+    ## tasks
+    task align {
+        Array[File] fastqs
+        String endedness
+        String aligner
+        File index
+        String? indexdir
+        String? libraryid
+        String bamroot
+        Int ncpus
+        Int ramGB
+        String? disks
+        Int? preemptible
+
+        command {
+            python3 $(which align.py) \
+                ${if length(fastqs)<2 then "--fastqs " + fastqs[0] else "--fastqs " + fastqs[0] + " " + fastqs[1]} \
+                --endedness ${endedness} \
+                --aligner ${aligner} \
+                --index ${index} \
+                ${"--indexdir " + indexdir} \
+                ${"--libraryid " + libraryid} \
+                ${"--bamroot " + bamroot} \
+                ${"--ncpus " + ncpus} \
+                ${"--ramGB " + ramGB}
+        }
+
+        output {
+            File genomebam = glob("*_genome.bam")[0]
+            File annobam = glob("*_anno.bam")[0]
+            File genome_flagstat = glob("*_genome_flagstat.txt")[0]
+            File anno_flagstat = glob("*_anno_flagstat.txt")[0]
+            File log = glob("*_Log.final.out")[0]
+            File python_log = glob("align.log")[0]
+        }
+
+        runtime {
+          cpu: ncpus
+          memory: "${ramGB} GB"
+          disks : select_first([disks, "local-disk 100 SSD"])
+          docker : "quay.io/encode-dcc/rna-seq-pipeline:v1.0"
+          preemptible: select_first([preemptible, 5])
+          bootDiskSizeGb: 10
+          noAddress: false
+        }
+    }
+
+    task bam_to_signals {
+        File input_bam
+        File chrom_sizes
+        String strandedness
+        String bamroot
+        Int ncpus
+        Int ramGB
+        String? disks
+        Int? preemptible
+
+
+        command {
+            python3 $(which bam_to_signals.py) \
+                --bamfile ${input_bam} \
+                --chrom_sizes ${chrom_sizes} \
+                --strandedness ${strandedness} \
+                --bamroot ${bamroot}
+        }
+
+        output {
+            Array[File] unique = glob("*niq.bw")
+            Array[File] all = glob("*ll.bw")
+            File python_log = glob("bam_to_signals.log")[0]
+        }
+
+        runtime {
+            cpu: ncpus
+            memory: "${ramGB} GB"
+            disks : select_first([disks,"local-disk 100 SSD"])
+            docker : "quay.io/encode-dcc/rna-seq-pipeline:v1.0"
+            preemptible: select_first([preemptible, 5])
+            bootDiskSizeGb: 10
```

Comment:
Isn't 10 the default for bootDisk?

Reply:
It might be, but I'm tempted to leave it explicit in case of future default changes. It also might have been necessary for ENCODE when running on different cloud infrastructures, so that could be helpful for us too?